infrastructure/ci/build-docker-images.sh

```diff
@@ -3,33 +3,17 @@
 # tagged with the build ID to DockerHub.
 set -ex
 
+echo "Logging into DockerHub"
+docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
```

Comment:
```suggestion
echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin
```
removes an anoying warning